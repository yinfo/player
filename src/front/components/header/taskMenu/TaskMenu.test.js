// @flow

import React from 'react';
import { shallow } from 'enzyme';
import TaskMenu from './TaskMenu';

describe('TaskMenu component', () => {
  it('renders as expected', () => {
    const component = shallow(<TaskMenu />);
    expect(component).toMatchSnapshot();
  });
});
