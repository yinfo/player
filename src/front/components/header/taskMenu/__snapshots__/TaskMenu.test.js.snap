// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaskMenu component renders as expected 1`] = `ShallowWrapper {}`;
